src/components/projects/fashion-editorial/FashionEditorial.tsx

```diff
@@ -1,19 +1,197 @@
-import FashionEditorialContent from "./FashionEditorialContent";
+import { LayoutProps, layout } from "styled-system";
+
 import FashionEditorialPreview from "./FashionEditorialPreview";
 import Flex from "../../Flex";
+import Grid from "../../Grid";
 import PreviewOrProjectPage from "../PreviewOrProjectPage";
 import React from "react";
+import image1 from "../../../assets/project-page/fashion-editorial/001-awewave-oxymore.jpg";
+import image10 from "../../../assets/project-page/fashion-editorial/010-awewave-oxymore.jpg";
+import image11 from "../../../assets/project-page/fashion-editorial/011-awewave-oxymore.jpg";
+import image12 from "../../../assets/project-page/fashion-editorial/012-awewave-oxymore.jpg";
+import image13 from "../../../assets/project-page/fashion-editorial/013-awewave-oxymore.jpg";
+import image14 from "../../../assets/project-page/fashion-editorial/014-awewave-oxymore.jpg";
+import image15 from "../../../assets/project-page/fashion-editorial/015-awewave-oxymore.jpg";
+import image16 from "../../../assets/project-page/fashion-editorial/016-awewave-oxymore.jpg";
+import image17 from "../../../assets/project-page/fashion-editorial/017-awewave-oxymore.jpg";
+import image18 from "../../../assets/project-page/fashion-editorial/018-awewave-oxymore.jpg";
+import image19 from "../../../assets/project-page/fashion-editorial/019-awewave-oxymore.jpg";
+import image2 from "../../../assets/project-page/fashion-editorial/002-awewave-oxymore.jpg";
+import image20 from "../../../assets/project-page/fashion-editorial/020-awewave-oxymore.jpg";
+import image21 from "../../../assets/project-page/fashion-editorial/021-awewave-oxymore.jpg";
+import image22 from "../../../assets/project-page/fashion-editorial/022-awewave-oxymore.jpg";
+import image3 from "../../../assets/project-page/fashion-editorial/003-awewave-oxymore.jpg";
+import image4 from "../../../assets/project-page/fashion-editorial/004-awewave-oxymore.jpg";
+import image5 from "../../../assets/project-page/fashion-editorial/005-awewave-oxymore.jpg";
+import image6 from "../../../assets/project-page/fashion-editorial/006-awewave-oxymore.jpg";
+import image7 from "../../../assets/project-page/fashion-editorial/007-awewave-oxymore.jpg";
+import image8 from "../../../assets/project-page/fashion-editorial/008-awewave-oxymore.jpg";
+import image9 from "../../../assets/project-page/fashion-editorial/009-awewave-oxymore.jpg";
+import styled from "styled-components";
+
+const Container = styled(Grid)<LayoutProps>`
```

Comment:
`ImageContainer`?

Reply:
i would move these styled components below the long array of images and its type definition so that its easier to have all your logic together rather than it being split by a bit array and having to scroll up and down the page a lot

Reply:
yah good shout

Reply:
+ just removed the unused prop `layout` from `Img`

Reply:
oh and from `ImgContainer`